templates/managed-rds-postgresql-v1.j2

```diff
@@ -0,0 +1,185 @@
+  DBBackupRetentionPeriod:
+    Description: The number of days for which automated backups are retained. Allowed
+      values 0-10. Setting this parameter to a positive number enables backups. Setting
+      this parameter to 0 disables automated backups.
+    Default: '1'
+    Type: Number
+    MinValue: '0'
+    MaxValue: '10'
+    ConstraintDescription: must be a number between 0 and 10.
+  VolumeSize:
+    Default: '20'
+    Description: General Purpose (SSD) storage (gp2). Must be an integer from 20 to
+      16384.
+    Type: Number
+    MinValue: '20'
+    MaxValue: '16384'
+    ConstraintDescription: Must be an integer from 20 to 16384.
+  StorageType:
+    Default: 'standard'
+    Description: Type of storage on the instance, whether standard ssd or optimized
+    Type: String
+    AllowedValues:
+      - standard
+      - gp2
+      - io1
+  EncryptVolume:
+    Default: 'true'
+    Description: Local encryption on the instance
+    Type: Boolean
+  DBEngineVersion:
+    Description: The database engine version
+    Type: String
+    Default: '11.2'
+    AllowedValues:
+      - '11.5R1'
+      - '11.2'
+      - '11.1'
+  InstanceType:
+    Description: The database instance type
+    Type: String
+    Default: db.t3.small
+    AllowedValues:
+      - db.t3.micro
+      - db.t3.small
+      - db.t3.medium
+      - db.t3.large
+      - db.t3.xlarge
+      - db.r4.large
+      - db.r4.xlarge
+      - db.m5.xlarge
+      - db.m5.large
+    ConstraintDescription: must select a valid database instance type.
+  MultiAZ:
+    Description: Multi-AZ database
+    Type: String
+    Default: 'true'
+    AllowedValues:
+      - 'true'
+      - 'false'
+    ConstraintDescription: must be true or false.
+Conditions:
+  HasMksKeyId: !Not [!Equals ["", !Ref KmsKeyId]]
+Resources:
+  DBSubnetGroup:
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupName: !Ref 'VpcName'
+      DBSubnetGroupDescription: Subnet group name within accounts
+      SubnetIds: !Ref 'VpcSubnet'
+  {% if sceptre_user_data.IngressWhitelist is defined %}
+  DBInstanceSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: "Open database on defined port for access by whitelisted IP"
+      VpcId: !ImportValue 'Fn::Sub': '${AWS::Region}-${VpcName}-VPCId'
+      SecurityGroupIngress:
+      {% for CIDRBlock in sceptre_user_data.IngressWhitelist %}
+        - IpProtocol: tcp
+          FromPort: !Ref 'DBPortNumber'
+          ToPort: !Ref 'DBPortNumber'
+          CidrIp: {{ CIDRBlock }}
+      {% endfor %}
+      SecurityGroupEgress:
+        - CidrIp: "0.0.0.0/0"
+        FromPort: -1
+        ToPort: -1
+        IpProtocol: "-1"
+      Tags:
+        - Key: "Department"
+          Value: !Ref Department
+        - Key: "Project"
+          Value: !Ref Project
+        - Key: "OwnerEmail"
+          Value: !Ref OwnerEmail
+  {% endif %}
+  DBInstance:
+    Type: 'AWS::RDS::DBInstance'
+    Properties:
```

Comment:
What happened to my suggestion of adding CopyTagsToSnapshot property?  I could have sworn i made this suggestion in one of my earlier reviews... https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-copytagstosnapshot

Reply:
I don't remember that in prior reviews, but it makes sense.